routes/api.php

```diff
@@ -1,8 +1,10 @@
 <?php
 
-// use Illuminate\Http\Request;
-// use Illuminate\Support\Facades\Route;
-//
-// Route::middleware('auth:api')->get('/user', function (Request $request) {
-//    return $request->user();
-// });
+// Articles
+use App\Http\Controllers\Articles\ArticlesController;
+
+Route::prefix('articles')->middleware('throttle:api')->group(function () {
```

Comment:
@lukeraymonddowning sorry a bit late to the party, I just noticed this great PR and wanted to shoot a video about it as an example PR. But am I missing something, or there should be also `auth:sanctum` middleware here? Cause, as it stands, there's no Sanctum "activated" for ArticlesController anywhere, or, again, maybe I'm missing something?

Reply:
Hey Povilas!

The auth middleware is actually in the Controller constructor in Laravel IO. Because of how Laravel's Auth middleware works, it will loop through each driver until it finds one that is acceptable. Hope that makes sense?

Reply:
@lukeraymonddowning ohhhh... I forgot to check in the global Controller. Sorry! And thanks for the clarification.